### PR TITLE
remove download report from compliance

### PR DESCRIPTION
**Pull request by ryelo**
## What

Removes download report from compliance

### CODE REVIEW

File: src/SmartComponents/Compliance/ComplianceCard.scss [hunk 1]
```diff
@@ -40,4 +40,8 @@
     > .pf-l-stack .pf-l-stack__item .pf-l-split .pf-l-split__item {
       font-size: var(--pf-global--FontSize--sm);
     }
-  }
\ No newline at end of file
+  }
+
+.ins-c-card__compliance {
```
Comment (AllenBW) [inline src/SmartComponents/Compliance/ComplianceCard.scss:45]: does this visually change the card?  should we apply this style to the template card (when sans actions?)

Comment (ryelo) [reply src/SmartComponents/Compliance/ComplianceCard.scss:45]: Yeah, when there is no report button on this card or remediations, extra padding is applied. We could probably extract it somewhere. @christiemolloy and ideas on where to add it?

Comment (christiemolloy) [reply src/SmartComponents/Compliance/ComplianceCard.scss:45]: umm id have to see it, want me to take this issue? @ryelo

Comment (ryelo) [reply src/SmartComponents/Compliance/ComplianceCard.scss:45]: Sure thing. You'll notice this in both remediations and compliance if you remove the TemplateCardActions

Comment (christiemolloy) [reply src/SmartComponents/Compliance/ComplianceCard.scss:45]: ok!

File: src/SmartComponents/Compliance/ComplianceCard.js [hunk 2]
```diff
@@ -64,7 +63,6 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
             } : { 'data-ouia-safe': false } }
         >
             <TemplateCardHead>
```
Comment (christiemolloy) [inline src/SmartComponents/Compliance/ComplianceCard.js:65]: @ryelo i made changes and tried to push them up but its not working. Just remove the `<TemplateCardHead>` wrapper, and then also the styles you added

Comment (ryelo) [reply src/SmartComponents/Compliance/ComplianceCard.js:65]: Awesome! Will do!

**Pull request merged**